samples/v2/instance.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2024 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// creates an instance using Instance Admin Client
+async function createInstance(instanceID, projectID) {
+  // [START spanner_create_instance]
+
+  // Imports the Google Cloud client library
+  const {InstanceAdminClient} = require('@google-cloud/spanner/build/src/v1');
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   **/
+  // const projectId = 'my-project-id';
+  // const instanceId = 'my-instance';
+
+  // Creates a client
+  const instanceAdminClient = new InstanceAdminClient({
+    projectId: projectID,
+  });
+
+  // Creates a new instance
+  try {
+    console.log(
+      `Creating instance ${instanceAdminClient.instancePath(
+        projectID,
+        instanceID
+      )}.`
+    );
+    const [operation] = await instanceAdminClient.createInstance({
+      instanceId: instanceID,
+      instance: {
+        config: instanceAdminClient.instanceConfigPath(
+          projectID,
+          'regional-us-central1'
+        ),
+        nodeCount: 1,
+        displayName: 'Display name for the instance.',
+        labels: {
+          ['cloud_spanner_samples']: 'true',
+          created: Math.round(Date.now() / 1000).toString(), // current time
+        },
+      },
+      parent: instanceAdminClient.projectPath(projectID),
```

Comment:
nit: I think it would be better to have this line at the top together with the `instanceId`